packages/guides-restructured-text/src/RestructuredText/Parser/Productions/FieldList/AbstractFieldListItemRule.php

```diff
@@ -17,7 +19,7 @@ public function applies(FieldListItemNode $fieldListItemNode): bool
         return strtolower($fieldListItemNode->getTerm()) === 'abstract';
     }
 
-    public function apply(FieldListItemNode $fieldListItemNode): MetadataNode
+    public function apply(FieldListItemNode $fieldListItemNode, DocumentNode $documentNode, DocumentParserContext $documentParserContext): MetadataNode
```

Comment:
You can get the Document From the DocumentParserContext. There is no need to add is separately.